src/app/shared/components/expenses-card-v2/expenses-card.component.ts

```
@@ -271,25 +230,44 @@
   }
 
-  ngOnInit(): void {
-    this.setupNetworkWatcher();
-    const orgSettings$ = this.orgSettingsService.get().pipe(shareReplay(1));
-
-    this.isSycing$ = this.isConnected$.pipe(
-      map((isConnected) => isConnected && this.transactionOutboxService.isSyncInProgress() && this.isOutboxExpense)
-    );
-
+  setExpenseDetails(): void {
     this.category = this.expense?.category?.name && this.expense?.category?.name.toLowerCase();
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)